hera_qm/xrfi.py

```diff
@@ -70,14 +70,32 @@ def flag_xants(uv, xants, inplace=True):
         return uvo
 
 
+def resolve_xrfi_path(xrfi_path, file):
+    """ Determine xrfi_path based on given directory or default to dirname of given file.
+    Args:
+        xrfi_path (str): Directory to write xrfi outputs.
+        file (str): Filename to determine backup directory if xrfi_path == ''
+    Returns:
+        dirname (str): If xrfi_path != '', returns xrfi_path. Otherwise returns
+            directory of file.
+    """
+    if xrfi_path != '':
+        dirname = xrfi_path
+    else:
+        dirname = os.path.dirname(os.path.abspath(file))
+    return dirname
+
+
 #############################################################################
 # Functions for preprocessing data prior to RFI flagging
 #############################################################################
 
-def medmin(d):
+def medmin(d, flags=None):
```

Comment:
To keep the interface to these functions uniform, I added a `flags` keyword to all of them, even if it's not used in most of them.